packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -1130,6 +1131,7 @@ export function useAutocomplete(props) {
       tabIndex: -1,
       type: 'button',
       onClick: handlePopupIndicator,
+      onMouseDown: (event) => event.stopPropagation(),
```

Comment:
Why this change?

Reply:
When the indicator was clicked, the `mouseDown event` of the root props triggered, causing the `handleOpen` function to execute. Subsequently, the intended indicator event occurred, resulting in the `handleClose` function being triggered.
To address this, I prevented the [propagation of the `mouseDown event` on the indicator](https://github.com/mui/material-ui/blob/8eef03cb64432f7e7b96c5b5459db8ee8f0da448/packages/mui-material/src/useAutocomplete/useAutocomplete.js#L1134).

Reply:
bug is [this comment](https://github.com/mui/material-ui/pull/42494#issuecomment-2216405861)